deploy/docker/Dockerfile

```diff
@@ -1,28 +1,44 @@
 FROM alpine:3.5
 
-LABEL maintainer="adundovi@gmail.com"
-LABEL description="Prepares a docker image \
-of pump.io server, http://pump.io/."
+LABEL maintainer Andrej Dundovic <adundovi@gmail.com>
+LABEL description "Stream server that does \
+most of what people really want from a social network."
 
 ENV LC_ALL en_US.UTF-8
 
+ENV PUMPIO_VERSION 3.0.2
```

Comment:
Responding to https://github.com/pump-io/pump.io/pull/1307#discussion_r109278471 - the point of maintaining it in the same repository is that we can update the Dockerfile at the same time we introduce new dependencies. So your second point isn't relevant, and we'd already have a cloned version of `package.json`.